cmd/flow-aggregator/flow-aggregator.go

```diff
@@ -15,6 +15,7 @@
 package main
 
 import (
+	"antrea.io/antrea/pkg/log"
```

Comment:
sorry I missed this: this import is in the wrong group

Reply:
Yup, I also just realized it. Updated the patch.